M2/Macaulay2/packages/Macaulay2Doc/doc5.m2

```diff
@@ -360,7 +360,7 @@ document { Key => {(eagonNorthcott,Matrix),eagonNorthcott},
 
 document { Key => {(selectVariables,List,PolynomialRing),selectVariables},
      Headline => "make a subring of a polynomial ring generated by selected variables",
-     Usage => "(S,f) = selectVariables(v,R)",
+     Usage => "(S,F) = selectVariables(v,R)",
```

Comment:
Was this change included by accident? `f` makes a lot more sense for a ring map than `F`.

Reply:
one of the variables is called `f`

Reply:
Ah .. still, maybe changing ring variables to x_1..x_8 would be better, but it's fine.